bin/helm_toolbox.sh

```diff
@@ -65,7 +59,7 @@ if [ "$1" == "upsert" ]; then
 	check helm
 	check timeout
 	set_context
-	upsert
+	retry upsert $RETRIES
```

Comment:
suggest using `retry $RETRIES upsert` instead. That way, if we ever add arguments to `upsert`, they can follow.